### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by YuliaHladyshkevych**

### CODE REVIEW

**Review by SevKrok — APPROVED** [review]
all good

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+        if isinstance(other, (int, float)):
+            return self.km > other
+        else:
+            return self.km > other.km
+
+    def __ge__(self, other: int | float | Distance) -> bool:
+        if isinstance(other, (int, float)):
+            return self.km >= other
+        else:
+            return self.km >= other.km
```
Comment (fsocie7y) [inline app/main.py:51]: u can do it without else)

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,63 @@
...
+        if isinstance(other, (int, float)):
+            return self.km < other
+        else:
+            return self.km < other.km
+
+    def __le__(self, other: int | float | Distance) -> bool:
+        if isinstance(other, (int, float)):
+            return self.km <= other
+        else:
+            return self.km <= other.km
```
Comment (fsocie7y) [inline app/main.py:63]: try to rewrite these two methods in one line each. Use previously created magic methods
Hint: <= is the same as not >

**Review by fsocie7y — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1,3 +1,54 @@
...
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | Distance) -> Distance:
```
Comment (LiudmylaKulinchenko) [inline app/main.py:15]: What if `other` is float here?

File: app/main.py [hunk 4]
```diff
@@ -1,3 +1,54 @@
...
+        return Distance(
+            self.km + other
+            if isinstance(other, (int, float))
+            else self.km + other.km
+        )
```
Comment (LiudmylaKulinchenko) [inline app/main.py:20]: Better not to use ternary operator here to increase readability and have consistent style throughout the code

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

**Review by LiudmylaKulinchenko — APPROVED** [review]
Great! 🐣

**Pull request closed without merging**